src/StackExchange.Redis/RedisFeatures.cs

```diff
@@ -211,6 +220,11 @@ public RedisFeatures(Version version)
         /// </summary>
         public bool KeyTouch => Version >= v3_2_1;
 
+        /// <summary>
+        /// Does the server prefer 'replica' terminology?
+        /// </summary>
+        public bool PreferReplica => Version >= v5_0_0;
```

Comment:
Maybe `ReplicaCommands`? Since PreferReplica is another thing on command flags, just thinking out loud here.